yapapi/log.py

```diff
@@ -314,6 +314,9 @@ class SummaryLogger:
     # Total time waiting for the first proposal
     time_waiting_for_proposals: timedelta
 
+    # Enable no activity warning
+    no_activity_warning_enabled = os.getenv("YAPAPI_NO_ACTIVITY_WARNING_ENABLED", "1") == "1"
```

Comment:
if anything, this should be a parameter to `enable_default_logger`, not an environment variable